nix/shell.nix

```
@@ -13,6 +13,6 @@ pkgs.mkShell {
     (callPackage ./riscv-gnu-toolchain.nix { })
     verible
     black
-    llvmPackages_15.clangUseLLVM
+    llvmPackages_14.clangUseLLVM
```

```suggestion
    llvmPackages.clangUseLLVM
```
We could also just use the default Nix version (currently version 11.1) instead of forcing version 14.